src/components/MenuButton/MenuButton.tsx

```diff
@@ -94,10 +104,10 @@ export default class MenuButton extends React.Component<MenuButtonProps, {}> {
 
   @autobind
   private getIcon() {
-    const { icon } = this.props;
+    const { icon, iconSize = IconSize.LARGE } = this.props;
     const Icon = icon || MoreIcon;
 
-    return <Icon size={IconSize.LARGE} block={true} />;
+    return <Icon size={iconSize} block={true} />;
```

Comment:
isn't there CSS on the IconButton setting it to 20px by 20px? do we need to map the sizes in CSS to the icon size?

Reply:
Yeah we did. Will have to map each icon size enum member to a css class I guess?